Code/Framework/AzCore/AzCore/Memory/HphaSchema.h

```diff
@@ -5,91 +5,17 @@
  * SPDX-License-Identifier: Apache-2.0 OR MIT
  *
  */
-#ifndef AZ_HPHA_ALLOCATION_SCHEME_ALLOCATOR_H
-#define AZ_HPHA_ALLOCATION_SCHEME_ALLOCATOR_H
+#pragma once
 
 #include <AzCore/Memory/Memory.h>
 #include <AzCore/std/typetraits/aligned_storage.h>
 
+ // The HphaSchemaErrorDetection.cpp file test the HphaSchema
+ // implementation with the DEBUG_ALLOCATOR define set to true
+ // In order a second implementation of this class to exist
+ // without polluting the test with using namespace ::AZStd/::AZ
+ // The interface is placed into an inl file with no includes
```

Comment:
This could use a proofread.  Is it c opied from somewhere?

Reply:
Yeah, I'll fix it.
It is missing some commas and conjunctions.